sphinx/cmd/quickstart.py

```diff
@@ -222,10 +222,7 @@ def ask_user(d: Dict) -> None:
                       'selected root path.')))
         print(__('sphinx-quickstart will not overwrite existing Sphinx projects.'))
         print()
-        d['path'] = do_prompt(__('Please enter a new root path (or just Enter to exit)'),
-                              '', is_path)
```

Comment:
It is better to replace `is_path` validator by new one instead of removal the feature.

Reply:
Alright, I'll give that a try :)